### ISSUE TITLE
A11y_AzureDataStudioAAE_PG1_Overview(Dashboard)_DeveloperTool: No name property is provided for "Service endpoints" and "Server group nodes" table, neither the table header role is assigned for the respective tables.

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15752]
![15752_A11y_AzureDataStudioAAE_PG1_Overview(Dashboard)_DeveloperTool](https://user-images.githubusercontent.com/85863543/127596522-4be6b96c-b2b1-4fc7-99f9-76b297a116b1.png)
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#DevTools;#WCAG4.1.2;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "PG1" and open manage and tab to "Overview(Dashboard)" page
5. Tab to "Service endpoints" and "Server group nodes" table and hit "ctrl + shift + i" and check if it has name provided.
6. Tab to "Service endpoints" and "Server group nodes" table respective table headers and hit "ctrl + shift + i" and check if it has role provided.

### Actual:
No name property is provided for "Service endpoints" and "Server group nodes" table, neither the table header role is assigned for the respective tables.
similar issue is observed through out the application where ever table is present.

### Expected:
Name property and table header role to the "Service endpoints" and "Server group nodes" table should be assigned according to it's label.

### User Impact:
Screen Reader users will not get the proper information about the table and its table headers.

**Comment by krrish2021** [comment]
Name for the table is still not set.
@candiceye the issue still repro in in latest azure data studio @Charles-Gagnon Please reopen the bug
![Screenshot (179)](https://user-images.githubusercontent.com/85863543/150956633-132966aa-0e40-4246-bfa6-f054987e0221.png)

**Comment by candiceye** [comment]
The tables have names, see below.
Service endpoints:
![image](https://user-images.githubusercontent.com/22386690/154426967-7bd7498f-0b3f-49de-b603-2ce2897c767b.png)

Server group nodes:
![image](https://user-images.githubusercontent.com/22386690/154426911-a9f815cf-6003-4520-8342-3bc1beb25647.png)
